spawner/README.md

```diff
@@ -39,20 +27,24 @@ There are two ways to run the spawner.
 	$ sudo usermod -aG $DOCKER_GID bugswarm
 	$ sudo su bugswarm
 	```
-1. Make sure `bugswarm-db` is built and set up on the host.
-
-### Setting up environment
-1. Inside the container, pull the git repository
+	
+#### OR Run Docker Inside Docker
+1. Run the container with `--privileged` mode.
 	```sh
-	$ git pull
+	$ docker run --privileged -it bugswarm-spawner
 	```
-1. Add the credentials file
+1. Start docker daemon and build and run the database container.
 	```sh
-	$ cat > bugswarm/common/credentials.py
+	$ sudo dockerd &
+	$ cd database
+	$ docker build -t bugswarm-db .
+	$ cd ..
+	$ docker run -tid -p 27017:27017 -p 5000:5000 bugswarm-db
 	```
-1. Re-run `./provision.sh`
+
+### Setting up environment
+1. Inside the container, pull the git repository
 	```sh
-	$ ./provision.sh
+	$ git pull
 	```
-1. Continue with any other commands (e.g. `./run_reproduce_project.sh`)
-
+1. Proceed with the database setup and provisioning instructions described [here](https://github.com/BugSwarm/bugswarm).
```

Comment:
```suggestion
1. Proceed with the database setup and provisioning instructions described [here](../README.md).
```

Reply:
Now that we've already made merged the Spawner instructions into the main ones onto the dev repo, I'll move the clarity related changes I made to that branch.